inventory_management_api/config.py

```diff
@@ -0,0 +1,8 @@
+def configure_app(app):
+    app.config["REDIS_HOST"] = "localhost"
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/trulyfurqan/eshop-fastapi/pull/7#issuecomment-2226005304>#907ae7</a><span> - 07/12/2024, 05:29 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Hardcoding sensitive information such as database credentials and hostnames in the code can lead to security vulnerabilities.
</div>
<div id="fix">
<b>Fix:</b> Use environment variables to store sensitive information and retrieve them in the 'configure_app' function.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,8 +1,14 @@
  def configure_app(app):
 -    app.config["REDIS_HOST"] = "localhost"
 -    app.config["REDIS_PORT"] = 6379
 -    app.config["DB_HOST"] = "localhost"
 -    app.config["DB_USER"] = "user"
 -    app.config["DB_PASSWORD"] = "password"
 -    app.config["DB_NAME_1"] = "inventory_db1"
 -    app.config["DB_NAME_2"] = "inventory_db2"
 +    import os
 +    app.config["REDIS_HOST"] = os.getenv("REDIS_HOST", "localhost")
 +    app.config["REDIS_PORT"] = int(os.getenv("REDIS_PORT", 6379))
 +    app.config["DB_HOST"] = os.getenv("DB_HOST", "localhost")
 +    app.config["DB_USER"] = os.getenv("DB_USER", "user")
 +    app.config["DB_PASSWORD"] = os.getenv("DB_PASSWORD", "password")
 +    app.config["DB_NAME_1"] = os.getenv("DB_NAME_1", "inventory_db1")
 +    app.config["DB_NAME_2"] = os.getenv("DB_NAME_2", "inventory_db2")
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged